packages/websocket-client/package.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
cc @karliatto we will need to publish this. All good from your side in the package config?

Reply:
It is missing the `"files"` section. I imagine in this package should look like:

```
"files": [
    "lib"
]
```

Regarding the field `"exports"` we can add it in a follow up when we add `libESM` since we want to keep using `"publishConfig"` as well.

Reply:
Since this package is dependency of `blockchain-link` it will be automatically added to the release process in next release.

Reply:
tnx, posted a fixup fcd363dc44e48d757d64319fce7d9154de0373ca